### ISSUE TITLE
Java 9+ Windows Display Scaling and chart rendering

### ISSUE THREAD

**Issue by colomb001** [jfree/jfreechart#170]
With Java 9 adding support for HiDPI monitors and resolution scaling, chart rendering quality suffers greatly on monitors with non 100% scaling values under Windows 10. This can be alleviated by turning off ChartPanel double buffering. It would be nice to retain double buffering for ChartPanels and not have rendering issues with resolution scaling turned on.

With double buffering enabled: ![doubleBuffering](https://user-images.githubusercontent.com/12431065/93526223-c2b04380-f8fc-11ea-9041-713e4254f33c.png)

With double buffering disabled: ![noDoubleBuffering](https://user-images.githubusercontent.com/12431065/93526226-c348da00-f8fc-11ea-8c85-752dbf4e79f8.png)

**Comment by trashgod** [comment]
See also this forum [thread](http://www.jfree.org/forum/viewtopic.php?f=3&t=120436).

**Comment by colomb001** [comment]
> See also this forum [thread](http://www.jfree.org/forum/viewtopic.php?f=3&t=120436).

That thread contains two posts, both from me. Unfortunately, neither provide a solution.

**Comment by trashgod** [comment]
I though the link might be useful to others.

**Comment by DrBAXA** [comment]
I did get the same issue on 4k screens. 
The problem is that image buffer with enabled buffering is created with the size of logical resolution provided by Swing. Real screen resolution is higher. Without buffering renderer uses something like vector graphics and Swing can manage this to look good but when first rendered into the image it can't be properly upscaled to the higher resolution. 

I have solved this issue and created pull request #179 , however, it looks slightly like a hack so I'm not sure if it should be accepted.
